ED/protocol.html

```diff
@@ -513,7 +516,7 @@ <h3 property="schema:name skos:prefLabel">Terminology</h3>
                     <dd about="#agent" property="skos:definition">An agent is a person, social entity, or software identified by a URI; e.g., a WebID denotes an agent [<cite><a class="bibref" href="#bib-webid">WEBID</a></cite>].</dd>
 
                     <dt about="#owner" property="skos:prefLabel" typeof="skos:Concept"><dfn id="owner">owner</dfn></dt>
-                    <dd about="#owner" property="skos:definition">An owner is a person or a social entity that is considered to have the rights and responsibilities of a data storage. An owner is identified by a URI, and implicitly has control over all data in a storage. An owner is first set at storage provisioning time and can be changed.</dd>
+                    <dd about="#owner" property="skos:definition">An owner is a person or a social entity that is considered to have the rights and responsibilities of a storage. An owner is identified by a URI, and implicitly has control over all URIs in a storage. An owner is first set at storage provisioning time and can be changed.</dd>
```

Comment:
Nit that it is unclear what "control over all URIs" means. Is it control over the content identified by those URIs? Or something more tricky?

Reply:
I see. How about:

* "implicitly has control over all resources in a storage" - a bit more readable I think.
* "implicitly controls a storage" - simple but hopefully not conflated with the storage resource (root container).

Reply:
First one works well for me. Second is similarly ambiguous.